src/app/api/api_v1/endpoints/cameras.py

```diff
@@ -46,7 +51,10 @@ async def fetch_cameras(
     token_payload: TokenPayload = Security(get_jwt, scopes=[UserRole.ADMIN, UserRole.AGENT, UserRole.USER]),
 ) -> List[Camera]:
     telemetry_client.capture(token_payload.sub, event="cameras-fetch")
-    return [elt for elt in await cameras.fetch_all()]
+    all_cameras = [elt for elt in await cameras.fetch_all()]
+    if UserRole.ADMIN in token_payload.scopes:
+        return all_cameras
+    return [camera for camera in all_cameras if camera.organization_id == token_payload.organization_id]
```

Comment:
Not sure but can't we make a single list comprehension?
```python
return [elt for elt in await cameras.fetch_all() if UserRole.ADMIN in token_payload.scopes or elt.organization_id == token_payload.organization_id]
```

Reply:
it seems to me that it's more readable in three lines, no ?